src/modules/check/derivation-path.ts

```diff
@@ -182,56 +182,58 @@ function derivationPath(
       }
       ;(window as any).handleCustomInput = handleCustomInput
       ;(window as any).handleDerivationClick = handleDerivationClick
-      return {
-        heading: heading || 'Hardware Wallet Connect',
-        description:
-          description ||
-          `Make sure your ${wallet.name} is plugged in, ${
-            wallet.name === 'Ledger' ? 'and the Ethereum app is open, ' : ''
-          }then select a derivation path to connect your accounts:`,
-        eventCode: 'derivationPath',
-        html: derivationSelectHtmlString(wallet.name as string),
-        button: {
-          text: 'Connect',
-          onclick: async () => {
-            state.loading = true
-            const path =
-              state.dPath || derivationPaths[wallet.name as string][0].path
-            try {
-              const validPath = await wallet.provider.setPath(
-                path,
-                state.showCustomInput
-              )
-
-              if (!validPath) {
-                state.error = `${path} is not a valid derivation path`
+      if (wallet.name !== 'CoboVault') {
```

Comment:
Also, how do you like this change ? @aaronbarnardsound 
For Cobo Vault not supporting customize derivation path currently, this check looks a lit weird for us.